### DISCUSSION TITLE
Is there a size limit for reveal.js presentation?

### DISCUSSION THREAD

**Discussion by Luc-UMass** [quarto-dev/quarto-cli#7730] [Q&A] [answered]
### Description

I am building up long slides using reveal.js (using vscode).  This is now the second time I run in the following problem, annoyingly enough just a few slides short of the end of my class.....

 When my slides hit a certain size, roughly 200 pages (this time 227 pages), then they my file will stop compiling.  No error message, just an endless wait. If I remove some material it will compile fine.

**Comment by cscheid** [comment]
That shouldn't happen; it has to be a bug somewhere. We'll need to see the source code that's causing the failure for you, though.

**Comment by Luc-UMass** [comment]
Hi Carlos:

Thanks for the quick answer.

I am attaching two files, the second one (appended with -notworking)
contains two more slides at the every end of the file.
The first one compiles fine, the second one does not.

I had exactly the same issue working on my last class last term.

Best,

Luc Rey-Bellet
Mathematics and Statistics
UMass Amherst





On Mon, Nov 27, 2023 at 6:49 PM Carlos Scheidegger ***@***.***>
wrote:

> That shouldn't happen; it has to be a bug somewhere. We'll need to see the
> source code that's causing the failure for you, though.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/7730#discussioncomment-7686497>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A222NLQ6FU44GDUAA4G76ILYGURH7AVCNFSM6AAAAAA74ZC3XWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMOBWGQ4TO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Luc Rey-Bellet
Mathematics and Statistics
UMass Amherst
***@***.***

**Reply by cscheid** [reply]
Email attachments don't work on GitHub issues, unfortunately. Could you maybe upload them to a GitHub repository and share the link?

**Reply by Luc-UMass** [reply]
Uploaded the files and here is the link

https://github.com/Luc-UMass/reveal.js-slides-issues

**Reply by cscheid** [reply]
I can't render your good file:

> File stochastic-bib.bib not found in resource path

We do need to try and reproduce the issue on our side, so can you ensure the repository you shared reproduces both the problem and the non-problem?

**Reply by Luc-UMass** [reply]
Thanks for your patience... I have now added all dependent files on github: pictures, .bib, and .css files

**Reply by cscheid** [reply, marked as answer]
Thank you, I can repro it. I've opened [an issue to track](https://github.com/quarto-dev/quarto-cli/issues/7737).

**Reply by cscheid** [reply]
@Luc-UMass I think I have a simple work-around for you while we diagnose this bug. Can you try rendering it with `--no-execute-daemon`? Something like

```
$ quarto render 1-ProbabilityModels-notworking.qmd --to revealjs --no-execute-daemon
```

In my local setup, it seems to avoid the problem (which is not that the execution is taking long, but that somehow a larger file is triggering a hang in our communication between the processes that manage the execution of the notebook).

**Reply by cscheid** [reply]
@Luc-UMass We have a fix incoming. I'm really sorry you ran into this bug, but thank you for your report and reproducible test case!

**Comment by Luc-UMass** [comment]
Oh great! That was fast.  The workaround is working for me for now.

I will say thanks to you and to the quarto team, this is a really great
product.  Besides the slides for my classes, I am in the process of writing
a book using quarto and this makes it easy!

**Reply by cscheid** [reply]
We're thrilled to hear. Please don't hesitate to reach out to us over here or on email should you have questions/comments.

**Comment by Luc-UMass** [comment]
Dear Carlos:

I am having a new issue running my slides on reveal.js.  The {.smaller}
command seems to have no effect anymore either as global command or on
single slides.  I can easily reproduce the issue on any new slide (running
quarto on vscode).

Should I open a new issue on this?

Luc Rey-Bellet



On Tue, Nov 28, 2023 at 3:52 PM Carlos Scheidegger ***@***.***>
wrote:

> We're thrilled to hear. Please don't hesitate to reach out to us over here
> or on email should you have questions/comments.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/7730#discussioncomment-7697059>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A222NLTOUDNSXPTP3KRMLA3YGZFHRAVCNFSM6AAAAAA74ZC3XWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMOJXGA2TS>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Luc Rey-Bellet
Mathematics and Statistics
UMass Amherst
***@***.***

**Reply by cscheid** [reply]
Yes, please.
